Casks/pe-client-tools-2019.8.rb

```diff
@@ -0,0 +1,25 @@
+cask 'pe-client-tools-2019.8' do
+  case MacOS.version
+  when '10.13'
+    os_ver = '10.13'
+    version '19.8.4'
+    sha256 '2c47f8ef4789d1cdf52d4b9d1e9d8e333101286117ab27bf6b9659929a7776fa'
+  when '10.14'
+    os_ver = '10.14'
+    version '19.8.4'
+    sha256 '1256319f55863af4ca8afb0eb3939e247a18104f5a0fd4cd4b59146487d99ca1'
+  else
+    os_ver = '10.15'
+    version '19.8.4'
+    sha256 '41b83875ff013d3e6dc91f59b49708d4076666e619d010c7cbce4f5412d0e3fa'
+  end
+
+  depends_on macos: '>= :high_sierra'
+  url "https://pm.puppet.com/pe-client-tools/2019.8.0/#{version}/repos/apple/#{os_ver}/PC1/x86_64/pe-client-tools-#{version}-1.osx#{os_ver}.dmg"
```

Comment:
```suggestion
  url "https://pm.puppet.com/pe-client-tools/2019.8.4/#{version}/repos/apple/#{os_ver}/PC1/x86_64/pe-client-tools-#{version}-1.osx#{os_ver}.dmg"
```
I think?

Reply:
No, the first version number is always a `.0` for the urls we use here. This is generated off of the erb template and the rake task. You can see it working in https://travis-ci.com/github/puppetlabs/homebrew-puppet/jobs/472208216

Ex:
https://github.com/puppetlabs/homebrew-puppet/blob/7d72054f5cf2052e2b16c54ac18cc4c18bfc9341/Casks/pe-client-tools-2019.3.rb#L14
https://github.com/puppetlabs/homebrew-puppet/blob/7d72054f5cf2052e2b16c54ac18cc4c18bfc9341/Casks/pe-client-tools-2018.1.rb#L14

Reply:
Oh, hmm....because https://pm.puppetlabs.com/pe-client-tools/2019.8.3/19.8.4/repos/apple/10.14/PC1/x86_64/pe-client-tools-19.8.4-1.osx10.14.dmg for example has 2019.8.3 (from https://puppet.com/misc/pe-files/previous-releases/2019.8.3/)